ROS/osr_bringup/config/roboclaw_params.yaml

```
@@ -1,36 +1,36 @@
 roboclaw_mapping:
-  # gear ratio is approx 172, exact value is 171.79 though
+  # gear ratio is approx 172, exact value is 26.9 though
   drive_left_front:
     address: 130
     channel: M2
-    ticks_per_rev: 48
-    gear_ratio: 171.79
+    ticks_per_rev: 28
+    gear_ratio: 26.9
   drive_left_middle:
     address: 130
     channel: M1
-    ticks_per_rev: 48
-    gear_ratio: 171.79
+    ticks_per_rev: 28
+    gear_ratio: 26.9
   drive_left_back:
     address: 129
     channel: M2
-    ticks_per_rev: 48
-    gear_ratio: 171.79
+    ticks_per_rev: 28
+    gear_ratio: 26.9
   drive_right_back:
     address: 129
     channel: M1
-    ticks_per_rev: 48
-    gear_ratio: 171.79
+    ticks_per_rev: 28
+    gear_ratio: 26.9
   drive_right_middle:
     address: 128
     channel: M2
-    ticks_per_rev: 48
-    gear_ratio: 171.79
+    ticks_per_rev: 28
+    gear_ratio: 26.9
   drive_right_front:
     address: 128
     channel: M1
-    ticks_per_rev: 48
-    gear_ratio: 171.79
-  # gear ratio for the corner motors is 16/48 as the CPR is measured after the gearbox, but
+    ticks_per_rev: 28
+    gear_ratio: 26.9
+  # gear ratio for the corner motors is 16/28 as the CPR is measured after the gearbox, but
```

```suggestion
  # gear ratio for the corner motors is 16/48 as the CPR is measured after the gearbox, but
```
Think you got trigger happy on this. That number is related to the corner motors, and should stay as-is.